### PR TITLE
Added a search functionality

### PR DESCRIPTION
**Pull request by arjo129**
Thanks for the wonderful tool! I've been having trouble looking for my country in this graph. I have added a search functionality to make it easier.

### CODE REVIEW

File: vue-definitions.js [hunk 1]
```diff
@@ -625,6 +626,17 @@ let app = new Vue({
 
     },
 
+    search() {
```
[the remark this answers is not in the document]
Comment (aatishb) [reply vue-definitions.js:629]: Finally, I wonder if we should add a debounce function, similar to the examples [here](https://vuejs.org/v2/guide/computed.html) or [here](https://vuejs.org/v2/examples/index.html). Perhaps we can use the approach [here](https://davidwalsh.name/javascript-debounce-function) or [here](https://github.com/jashkenas/underscore/blob/master/underscore.js#L879-L910), and add a debounce method?

File: index.html [hunk 2]
```diff
@@ -119,14 +119,16 @@ <h2>Customize</h2>
           <h2>{{regionType}}</h2>
 
           <p>{{regionType}} with at least {{minCasesInCountry}} {{selectedData}}</p>
-
+          
+          
           <div id="buttonwrapper">
+            <div><input @change="search" v-model="searchField"></input></div>
```
Comment (aatishb) [inline index.html:125]: Let's add a placeholder text = 'Search' here, so the purpose of this input is clear. Like in https://vuejs.org/v2/guide/forms.html#Text

File: vue-definitions.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (arjo129) [reply vue-definitions.js:629]: If you want me to de-bounce the search, should I add a dependency on lo-dash like in the examples or do you want me to implement it separately?

File: vue-definitions.js [hunk 3]
```diff
@@ -772,10 +784,14 @@ let app = new Vue({
 
     countries: [],
 
+    visibleCountries: [], 
+
     isHidden: true,
 
     selectedCountries: [],
 
+    searchField: "",
```
[the remark this answers is not in the document]
Comment (arjo129) [reply vue-definitions.js]: Resolved in a71f5b3 .

File: index.html [hunk 4]
```diff
@@ -119,14 +119,16 @@ <h2>Customize</h2>
           <h2>{{regionType}}</h2>
 
           <p>{{regionType}} with at least {{minCasesInCountry}} {{selectedData}}</p>
-
+
```
[the remark this answers is not in the document]
Comment (arjo129) [reply index.html:122]: Resolved in a71f5b3

File: index.html (hunk 2 above)
Comment (arjo129) [reply index.html]: Resolved in a71f5b3

File: vue-definitions.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (arjo129) [reply vue-definitions.js:649]: Resolved in 063a7eb

**Comment by arjo129** [comment]
OK done with your suggested changes!

**Comment by aatishb** [comment]
Thanks @arjo129! Working on this now.

**Comment by aatishb** [comment]
Nice work @arjo129!

**Comment by aatishb** [comment]
@all-contributors add @arjo129 for code

**Pull request merged**